page.php

```diff
@@ -1,5 +1,32 @@
 <?php
 get_header();
+
+function render_submenu($post) {
```

Comment:
it was a consious decision i wasnt being lazy. but it is tidier to keep all in one place and i think this function actually depends on another custom function so that undermines my arg somewhat.

perhaps we should be even tidier and just use functions.php to include other files so we can organize that file a bit better. frankly our functions.php is ugly already